### PR TITLE
Added MiniMax config

### PR DESCRIPTION
**Pull request by lchoquel**
## Summary by cubic
Adds the `minimax` backend via MiniMax’s Anthropic-compatible API with M2-series presets and a routing profile. Enables manual thinking support and fixes model pricing; tests, CI env placeholders, and `kit` configs are synced.

- **New Features**
  - Added `minimax` backend at https://api.minimax.io/anthropic with M2, M2.1, M2.5, M2.7 (+ `-highspeed`) presets.
  - Routing profile `profiles.all_minimax` (default `minimax`).
  - Enabled thinking support with `thinking_mode = "manual"` (maps to Anthropic thinking params).
  - Pricing: standard models 0.3/1.2 USD per MTok (in/out); highspeed 0.6/2.4.
  - Tests/CI: include MiniMax in test profiles; clearer skip reasons for empty param sets; add `MINIMAX_API_KEY` placeholders.
  - Kit sync: mirrored backend and routing configs under `pipelex/kit/configs`.

- **Migration**
  - Set `MINIMAX_API_KEY` in your environment.
  - Select `profiles.all_minimax` or target the `minimax` backend directly.

<sup>Written for commit b2b21f04333634fa8206782eb979251e880653ad. Summary will update on new commits.</sup>

### CODE REVIEW

File: pipelex/kit/configs/inference/backends/minimax.toml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+################################################################################
+# LANGUAGE MODELS
+################################################################################
+
+# --- MiniMax M2 Series --------------------------------------------------------
+[minimax-m2]
+model_id = "MiniMax-M2"
+inputs = ["text"]
+outputs = ["text", "structured"]
+costs = { input = 0.27, output = 1.15 }
```
[the remark this answers is not in the document]
Comment (lchoquel) [reply pipelex/kit/configs/inference/backends/minimax.toml]: Fixed

File: pipelex/kit/configs/inference/backends/minimax.toml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+################################################################################
+# MODEL DEFAULTS
+################################################################################
+
+[defaults]
+model_type = "llm"
+sdk = "anthropic"
+prompting_target = "anthropic"
+structure_method = "instructor/anthropic_tools"
+thinking_mode = "none"
```
[the remark this answers is not in the document]
Comment (lchoquel) [reply pipelex/kit/configs/inference/backends/minimax.toml]: fixed
